manifests/check.pp

```diff
@@ -107,11 +107,18 @@
 #   Set this to 'absent' to remove it completely.
 #   Default: undef
 #
+# [*content*]
```

Comment:
This seems to be the same thing as custom, so I don't think we need it on this iteration.

Next iteration would be to remove all parameters and only have `content`. At that point we can use the Struct data type for validation.